### ISSUE TITLE
Camera bug when entering IVA.

### ISSUE THREAD

**Issue by NexxusDrako313** [icedown/KSP-ProbeControlRoom#6]
I'm attempting to enter the Probe Control Room, but always wind up in an awkward camera angle somewhere just outside the ship when pressing "C". I can use the right click menu on the probe core to enter the PCE, but it's inconvenient, as I'm trying to run a PCR only save for extra challenge.

**Comment by NexxusDrako313** [comment]
I'm thinking MFD Revived may be the cause of the problem. I've just removed it and will test again to check if it works.

**Comment by icedown** [comment]
Yes, that shows up when PCR has an issue.  It is the seat position of the internal camera  Please post your output_log.txt and I will look into it.

**Comment by NexxusDrako313** [comment]
Righty-o. I will note that my game is very heavily modded, so that could also be a problem.

https://my.pcloud.com/publink/show?code=XZI0llZQD3X5Ynl4eLrNOxfyEWqH7wEgXbX

Sorry for the 40,000kb text file, I 7zipped it down as small as I could for you. heh
